src/app/components/Shared/input.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export default function Input({name, id, placeholder}) {
+  return (
+    <input
+      type="text"
+      name={name}
+      id={id}
+      className="shadow-sm focus:ring-orange-bitcoin h-14 focus:border-orange-bitcoin block w-full sm:text-sm border-gray-300 rounded-md"
+      placeholder={placeholder}
+    />
+  );
+}
```

Comment:
I would only override the `className` property. Or even better extend it…

```suggestion
export default function Input({className, ...props}) {
  return (
    <input
      {...props} 
      className={`shadow-sm focus:ring-orange-bitcoin h-14 focus:border-orange-bitcoin block w-full sm:text-sm border-gray-300 rounded-md ${className}`}
    />
  );
}
```